tests/playbooks/tests_802_1x.yml

```diff
@@ -89,6 +89,50 @@
                   domain_suffix_match: example.com
         - name: "TEST: I can ping the EAP server"
           shell: ping -c1 203.0.113.1
+        - import_role:
+            name: linux-system-roles.network
+          vars:
+            network_connections:
+              - name: "{{ interface }}"
+                persistent_state: absent
+                state: absent
+        - name: >-
+            TEST: 802.1x profile with unencrypted private key and ca_path
+          debug:
+            msg: "##################################################"
+        - name: Create directory for ca_path test
+          file:
+            path: "/etc/pki/tls/my_ca_certs"
+            state: directory
+            mode: 0755
+        - name: Copy cacert to ca_path
+          copy:
+            src: "cacert.pem"
+            dest: "/etc/pki/tls/my_ca_certs/cacert.pem"
```

Comment:
You need to run `ln -s "cacert.pem" "$(openssl x509 -hash -noout -in "cacert.pem")".0` in that directory to hash the file - it is a one-liner to avoid installing c_rehash which requires perl, see: https://stackoverflow.com/questions/25889341/what-is-the-equivalent-of-unix-c-rehash-command-script-on-linux